jcdb-api/src/main/kotlin/org/utbot/jcdb/api/ext/KMetadataExtension.kt

```diff
@@ -44,6 +52,10 @@ val JcClassOrInterface.kMetadata: KotlinClassMetadata?
         return KotlinClassMetadata.read(kmHeader)
```

Comment:
there is no guarantee that this code will not fail. Let's wrap into try-catch and write a warning in logs about it

Reply:
Done